drivers/qemu/driver.go

```diff
@@ -234,7 +254,13 @@ func (d *Driver) buildFingerprint() *drivers.Fingerprint {
 		HealthDescription: drivers.DriverHealthy,
 	}
 
-	bin := "qemu-system-x86_64"
+	bin, err := resolveBinPath(d.binPath)
+	if err != nil {
+		fingerprint.Health = drivers.HealthStateUndetected
+		fingerprint.HealthDescription = err.Error()
```

Comment:
This makes the CLI `node status -self -verbose` output *really* wide. Should I shorten it to `QEMU binary not found` and just log the full error? Including the path makes it arbitrarily wide but makes debugging way easier than digging through client logs.

Example output with a *short* binary name:
```
Drivers
Driver       Detected  Healthy  Message                                                                    Time
docker       false     false    Failed to connect to docker daemon                                         2022-12-09T10:06:19-08:00
exec         false     false    Driver must run as root                                                    2022-12-09T10:06:19-08:00
java         false     false    Driver must run as root                                                    2022-12-09T10:06:19-08:00
mock_driver  true      true     Healthy                                                                    2022-12-09T10:06:19-08:00
qemu         false     false    unable to use binary_path="asdf-ARCH": executable file not found in $PATH  2022-12-09T10:06:19-08:00
raw_exec     true      true     Healthy                                                                    2022-12-09T10:06:19-08:00
```

Reply:
If `binary_path` is unset does it retain the `<none>` message? The verbosity when set seems fine - in that case you set bad config and should feel bad

Reply:
Oh that's a fantastic idea. I'll implement that, thanks.